storage/rocksdb/build_rocksdb.cmake

```diff
@@ -507,6 +507,10 @@ if(MSVC)
   # Workaround Win8.1 SDK bug, that breaks /permissive-
   string(REPLACE "/permissive-" "" CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS}")
 else()
+  set_source_files_properties(${ROCKSDB_SOURCE_DIR}/options/db_options.cc
```

Comment:
@dr-m @vlad-lesin  on the assumption that the two warnings in the commit message are what was seen I pushed the warning down two just these two files and perhaps like many of the other things in the file assumed clang or gcc.

Reply:
Could you add a couple of commas to your GitHub comment here?